.github/renovate.json

```diff
@@ -1,22 +1,54 @@
 {
   "extends": [
     "config:base",
-    ":semanticCommitTypeAll(chore)"
+    ":semanticCommitTypeAll(chore)",
+    ":disableDependencyDashboard"
+  ],
+  "ignorePresets": [
+    ":semanticPrefixFixDepsChoreOthers"
   ],
-  "ignorePresets": [":semanticPrefixFixDepsChoreOthers"],
   "prConcurrentLimit": 0,
-  "rebaseStalePrs": true,
+  "rebaseWhen": "behind-base-branch",
   "dependencyDashboard": true,
-  "semanticCommits": true,
+  "semanticCommits": "enabled",
   "postUpdateOptions": [
     "gomodTidy"
   ],
+  "packageRules": [
+    {
+      "description": "Disable MAJOR update types",
+      "matchUpdateTypes": [
+        "major"
+      ],
+      "enabled": false
+    }
+  ],
+
   "ignoreDeps": [
-      "golang.org/x/net"
+    "golang.org/x/net"
```

Comment:
You can remove this -- we'd added it for Go 1.16 I believe elsewhere.

Reply:
I'll do this in a separate PR.